.github/actions/custom-actions/aselo_development_custom/action.yml

```diff
@@ -1,18 +1,22 @@
 name: 'Aselo Development release custom action'
 description: 'Sets up environment variables related to non-standard additional features in use in the development environment'
+inputs:
+  account-sid:
+    description: 'The Twilio Account SID for this account'
+    required: true
 runs:
   using: "composite"
   steps:
     # Set IWF credentials
     - name: Set IWF_API_USERNAME
       uses: "marvinpinto/action-inject-ssm-secrets@latest"
       with:
-        ssm_parameter: "IWF_API_USERNAME"
+        ssm_parameter: IWF_API_PASSWORD
```

Comment:
I this change intended? Looks like it's setting local env var `IWF_API_USERNAME` with the value of `IWF_API_PASSWORD`.